internal/resources/metal/organization_member/resource_schema.go

```diff
@@ -0,0 +1,63 @@
+package organizationmember
+
+import (
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+func GetResourceSchema() *schema.Schema {
+	return &schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"invitee": schema.StringAttribute{
+				Description: "The email address of the user to invite",
+				Required:    true,
+				Validators: []validator.String{
+					stringvalidator.LengthAtLeast(1),
+				},
+			},
+			"invited_by": schema.StringAttribute{
+				Description: "The user id of the user that sent the invitation (only known in the invitation stage)",
+				Computed:    true,
+			},
+			"organization_id": schema.StringAttribute{
+				Description: "The organization to invite the user to",
+				Required:    true,
+				Validators: []validator.String{
+					stringvalidator.LengthAtLeast(1),
+				},
+			},
+			"project_ids": schema.SetAttribute{
```

Comment:
this field was in plural `projects_ids` instead of `project_ids` https://github.com/equinix/terraform-provider-equinix/blob/12c5b6b21a6d385cca5863d6c58a5a3702b6a012/equinix/resource_metal_organization_member.go#L77